microsoft-edge/devtools-guide-chromium/whats-new/2022/04/devtools-101.md

```diff
@@ -0,0 +1,122 @@
+
+To report a problem with DevTools or ask for a new feature, check out the new [MicrosoftEdge > DevTools](https://github.com/MicrosoftEdge/DevTools) repo.
+
+
+<!-- ====================================================================== -->
+## The Console tool is now available in Visual Studio Code
+
+<!-- Title: Visual Studio Code extension now with Console tool -->
+<!-- Subtitle: See log messages, run JavaScript, and use Console APIs right next to your code in Visual Studio Code. -->
+
+Using the Console is one of the biggest use-cases of DevTools.  That's why we made it easier for you to access the Console tool by turning it on in the Microsoft Edge Developer Tools extension for Visual Studio Code.  This allows you to see your log messages right next to your code.  You can also try out some JavaScript,<!--enter JavaScript code,--> access the window object, and use the Console API to access the DOM.  You can use the Console as the main tool, or use it together with the **Elements** and **Network** tools.
+
+The Console tool in the main toolbar of the Edge DevTools extension for Visual Studio Code:
+
+![The Console tool in the main toolbar of the Edge DevTools extension for Visual Studio Code.](devtools-101-images/console-tool-vsc-1.png)
+
+The Console tool in the Drawer of the Edge DevTools extension for Visual Studio Code:
+
+![The Console tool in the Drawer of the Edge DevTools extension for Visual Studio Code.](devtools-101-images/console-tool-vsc-2.png)
+
+For more information, see [Console integration](../../../../visual-studio-code/microsoft-edge-devtools-extension.md#console-integration) in _Microsoft Edge DevTools extension for Visual Studio Code_.
+
+
+<!-- ====================================================================== -->
+## Source maps are now cached with IndexedDB
+
+<!-- Title: Speed up debugging with cached sourcemaps -->
+<!-- Subtitle: DevTools now caches sourcemaps with IndexedDB, reducing the need to fetch sourcemaps on refresh. -->
+
+In previous versions of Microsoft Edge, if you were hosting large sourcemap files on your server or in [Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md), DevTools would fetch these sourcemaps every time the page refreshed.  Fetching these sourcemaps repeatedly created additional network requests for DevTools and slowed down debugging time.
+
+In Microsoft Edge 101, DevTools caches sourcemaps by using IndexedDB.  Upon refreshing the page, sourcemaps are now served from the IndexedDB cache instead of being requested again from your server or Azure Artifacts symbol server.  Use the **Load status** column in the [Source Maps Monitor tool](../../../source-maps-monitor/source-maps-monitor-tool.md) to verify that sourcemaps are being loaded from the cache.  To clear the IndexedDB cache and force DevTools to reload sourcemaps, run the **Clear source maps cache** command in the [Command Menu](../../../command-menu/index.md).
```

Comment:
```suggestion
In Microsoft Edge 101, DevTools caches sourcemaps by using [IndexedDB](https://developer.mozilla.org/docs/Web/API/IndexedDB_API).  Upon refreshing the page, sourcemaps are now served from the IndexedDB cache instead of being requested again from your server or Azure Artifacts symbol server.  Use the **Load status** column in the [Source Maps Monitor tool](../../../source-maps-monitor/source-maps-monitor-tool.md) to verify that sourcemaps are being loaded from the cache.  To clear the IndexedDB cache and force DevTools to reload sourcemaps, run the **Clear source maps cache** command in the [Command Menu](../../../command-menu/index.md).
```

Minor tweak.  Maybe some readers don't know what IndexedDB is and why it's useful so we could link to MDN optionally